tests/testCmd.py

```diff
@@ -56,7 +56,7 @@ def tearDown(self):
                 for file in subdirs:
                     os.rmdir(os.path.join(dir,file))
             os.rmdir(newprod)
-                    
```

Comment:
Whitespace-only changes make commits difficult to read. Could you fix these (ignore the comment if they're already on a commit of their own)?